From 3e6339e07bcd9e010bce14fa6b93526f55706d7f Mon Sep 17 00:00:00 2001
From: Jan Tulak <jtulak@redhat.com>
Date: Wed, 6 Mar 2019 15:30:45 +0100
Subject: [PATCH 10/22] ssmlib: fix sizes in xfs_get_info for mounted devices

ssm usually works with sizes in kb, but statvfs reports in bytes,
so the values we got for mounted xfs were off by an order. A simple
division by 1024 brings that into order.

RHBZ 1664652

Signed-off-by: Jan Tulak <jtulak@redhat.com>
---
 ssmlib/main.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ssmlib/main.py b/ssmlib/main.py
index 5c94c3e..355c0e8 100644
--- a/ssmlib/main.py
+++ b/ssmlib/main.py
@@ -327,9 +327,9 @@ class FsInfo(object):
         mount_point = misc.get_mounts(dev).get(realdev, {}).get('mp', None)
         if mount_point:
             stat = os.statvfs(mount_point)
-            total = stat.f_blocks*stat.f_bsize
-            free = stat.f_bfree*stat.f_bsize
-            used = (stat.f_blocks-stat.f_bfree)*stat.f_bsize
+            total = stat.f_blocks*stat.f_bsize/1024
+            free = stat.f_bfree*stat.f_bsize/1024
+            used = (stat.f_blocks-stat.f_bfree)*stat.f_bsize/1024
             self.data['fs_size'] = total
             self.data['fs_free'] = free
             self.data['fs_used'] = used
-- 
2.21.0

